t/test_cleanupSpies

```diff
@@ -0,0 +1,48 @@
+#!/bin/sh
+
+readonly TEST_DIR="$(dirname "$0")"
+
+# shellcheck source=/dev/null
+. "$TEST_DIR/../shpy"
+# shellcheck source=/dev/null
+. "$TEST_DIR/../shpy-shunit2"
+
+testRemovesDirectory() {
```

Comment:
The tests should follow a `testCause_Effect` naming scheme (which isn't documented anywhere, I'll look into adding a contributing.md)

```suggestion
testCleanupSpies_RemovesTmpDirectory() {
```